apis/python/tests/test_basic_anndata_io.py

```diff
@@ -856,8 +856,27 @@ def test_X_none(h5ad_file_X_none):
         assert exp.ms["RNA"].var.count == 1838
         assert list(exp.ms["RNA"].X.keys()) == []
 
-        tiledbsoma.io.to_anndata(exp, measurement_name="RNA", X_layer_name=None)
-        # TODO: more
+        adata = tiledbsoma.io.to_anndata(exp, measurement_name="RNA", X_layer_name=None)
```

Comment:
Do we need a test case to differentiate between not specifying an X_layer_name  and  specifying None as X_layer_name, when X.keys() contains `data`?